src/scripts.js

```diff
@@ -438,3 +442,58 @@ var stepTrend = new Chart(ctx, {
     }
   }
 });
+
+// *** EVENT LISTENERS FOR HEADER ***
+$('.toggle label').on('click', function() {
+  if ($(this).siblings().prop('checked')) {
+    changeMode('light');
+  } else {
+    changeMode('dark');
+  }
+  Chart.defaults.global.defaultFontColor = $('body').css('--base-color');
+});
+
+$('.icons li img').on('click', function() {
+  const $widget = $(this).data('type');
+  const $block = $('#user-admin-info');
+  if ($block.data('type') === $widget) {
+    $block.data('type', '').hide();
+  } else {
+    $block.show().children(`.${$widget}-inputs`).show().siblings().hide();
+    $block.data('type', $widget);
+  }
+});
+
```

Comment:
Could you walk me through what is happening here? You are making a jQuery obj out of the data('type')? This function is responsible for toggling the drop down?
I have seen it in action and it looks great but just need a little clarification. Thanks!

Reply:
Second that. Pretty sure this is for the user to input new data by clicking on one of the icons in the right corner of the header. So as we click one child will appear and we hide all the siblings. If the user mouses out of the form field then the form hides again.